### ISSUE TITLE
Build issues, due to SVN library changes

### ISSUE THREAD

**Issue by philstopford** [ChaosGroup/blender_with_vray_additions#267]
It looks like the blender SVN library repository has removed several required libraries (e.g. pthread) that break the build of the V-ray/blender code.

**Comment by poseidon4o** [comment]
Yes, please use revision 62167, this is what  the nightly builds are using.
